vars.tf

```diff
@@ -18,6 +18,12 @@ variable "vpc_cidr" {
   }
 }
 
+variable "subnet_cidrs_offset" {
+  type        = number
+  default     = 1
```

Comment:
Why `1`? 
`0` would give the same as before right?

Reply:
My mistake. 😊 Fixed!